packages/framework-core/src/sensor/health/health-indicators/booster-database-readmodels-catalogs-health-indicator.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  BoosterConfig,
+  HealthIndicatorResult,
+  HealthIndicatorMetadata,
+  HealthStatus,
+} from '@boostercloud/framework-types'
+
+export class BoosterDatabaseReadmodelsCatalogsHealthIndicator {
```

Comment:
What does `Catalogs` in the name refers to?

Reply:
It refers to the metadata in which definitions of tables are defined. The idea is that indicator will provide this information, not the table contents. Any suggestion to improve it will be welcomed

Reply:
@gonzalogarciajaubert Mmm mayby I still don't get it, but to me it can be removed and leave the class name as `BoosterDatabaseReadmodelsHealthIndicator`

What do you think

Reply:
Yeah, it could be confusing some times but let me change it and we could review it.